packages/sdk/src/utils/Mapping.ts

```diff
@@ -19,6 +19,14 @@ interface Item<K, V> {
     value: V
 }
 
+interface LRU<K extends string, V> {
```

Comment:
Could this pick something like `type AbstractMap = Pick<Map, ...>`. In the `else` branch we create an instance, which is not a LRU (just a simple Map) and therefore could rename e.g. to `AbstractMap`.

Reply:
Types of `Map` and `LRUCache` are incompatible. I'll keep the explicit matching-all type and change its name to `CacheMap`.